webview-ui/src/components/ResolutionsPage/ResolutionsPage.tsx

```diff
@@ -78,23 +76,14 @@ const ResolutionPage: React.FC = () => {
       </PageSection>
 
       <PageSection>
-        <Flex
-          direction={{
-            default: "column",
-          }}
-        >
+        <Flex direction={{ default: "column" }} className="chat-container">
           {isTriggeredByUser && (
             <Flex
-              direction={{
-                default: "column",
-              }}
+              direction={{ default: "column" }}
               grow={{ default: "grow" }}
               alignItems={{ default: "alignItemsFlexEnd" }}
-              justifyContent={{ default: "justifyContentFlexEnd" }}
             >
-              <FlexItem>
-                <YellowLabel>Here is the scope of what I would like you to fix:</YellowLabel>
-              </FlexItem>
+              <SentResponse>Here is the scope of what I would like you to fix:</SentResponse>
               <FlexItem className="chat-card-container">
                 <ChatCard color="yellow">
```

Comment:
ChatCard is no longer yellow but the rest of user triggered messages is still yellow. We should be consistent here. If keeping the yellow background (even) for light theme is complicated then lets drop it and use system background. Fortunately the blue background seems universal so we maintain visual difference with the server side messages.